src/workflows/airqo_etl_utils/bigquery_api.py

```
@@ -1279,10 +1279,16 @@ def fetch_device_data_for_forecast_job(
             ValueError: If the provided start_date_time is invalid.
             RuntimeError: If there is an error fetching data from BigQuery.
         """
+        valid_job_types = {"train", "predict"}
+        if job_type not in valid_job_types:
+            raise ValueError(
+                f"Invalid job_type '{job_type}'. Expected one of {sorted(valid_job_types)}."
+            )
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)